modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -365,17 +380,16 @@ sda.enkf.multisite <- function(settings,
         #for restart when t=1 do not need to do model runs and X should already exist in environment by this point
         X <- X
       }else{
-        
         if (control$debug) browser()
         #if restart then use restart.list, include site for debugging purposes
         if(restart_flag){
-          restart.arg = restart.list #$`646`debugging hack site specific
+          restart.arg = restart.list$`646`
         }else{
           restart.arg = NULL
         }
         out.configs <- conf.settings %>%
           `class<-`(c("list")) %>%
-          furrr::future_map2(restart.list, function(settings, restart.arg = restart.arg) {
+          furrr::future_map2(restart.list, function(settings, restart.arg) {
```

Comment:
not sure if this is new or old, but curious why the default was removed

Reply:
I think it's meaningless to have a default here cause the `restart.arg` has to be pointed to the subset of the `restart.list`